.eslintrc.json

```diff
@@ -0,0 +1,24 @@
+{
+    "env": {
+      "browser": true,
+      "es2021": true,
+      "node": true
+    },
+    "extends": "eslint:recommended",
+    "overrides": [
+      {
+        "files": ["**/*.cy.js"],
```

Comment:
Jest config is missing here, you should have a second override object with similar rules but for jest.